ios/MaasCore/Sources/MaasCore/MaasCore.swift

```diff
@@ -1 +1 @@
-@_exported import MaasCore
+@_exported import CoreBinary
```

Comment:
I am fascinated how these `@_exported import`s interact with SPM 🤔 Without this change, Xcode complains about a cyclical dependency between `MaasCore` & `MaasComponents`.

Reply:
This will be solved in this pull request. Looks like in `main` we have a `.xcframework` which is named not a `CoreBinary` but `MaasCore`.